src/styles/blocks/card.scss

```
@@ -0,0 +1,110 @@
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+
+  font-family: "Roboto", sans-serif;
+  font-size: 0;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__piсture {
+    padding-left: 3px;
+    padding-right: 3px;
+    margin-bottom: 40px;
+  }
+
+  &__img {
+    max-width: 100%;
+    height: 134px;
+  }
+
+  &__title {
+    display: inline-block;
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__sub-title {
+    display: inline-block;
+    margin-bottom: 16px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    align-content: center;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    display: inline-block;
+    align-self: flex-end;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+    color: #060b35;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: #060b35;
+  }
```

ніби ось так можна скоротити, але я не впевнений чи ми змушені це по таску робити
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-bottom: 16px;
    
    &-text {
      font-weight: 400;
      font-size: 12px;
      line-height: 18px;
      color: #616070;
    }
    
    &-value {
      font-weight: 700;
      font-size: 16px;
      line-height: 18px;
      color: #060b35;
  }
  }

  

  
```

---

Думаю, что `&__price` это элемент и  `&__price-text`  это элемент, а не блок и понятнее в коде будет `&__price-text`.
Что будет, если захочу применить camelCase:
  `price-text` =>  `priceText`

---

Всё верно, не нужно использовать & чтобы собирать слова по частям 🙂 Лучше использовать только для элементов, модификаторов, псевдоэлементов и псевдоклассов